tracing/trace_keyspace_helper.cc

```diff
@@ -289,10 +289,23 @@ cql3::query_options trace_keyspace_helper::make_slow_query_mutation_data(const o
     const session_record& record = session_records.session_rec;
     auto millis_since_epoch = std::chrono::duration_cast<std::chrono::milliseconds>(record.started_at.time_since_epoch()).count();
 
-    // query command is stored on a parameters map with a 'query' key
+    sstring query_str;
+    // query command is stored on a parameters map with a 'query' key or 'query[N]' keys
     auto query_str_it = record.parameters.find("query");
```

Comment:
Super nit: This variable could be defined in smaller scope now (if scope), no?

Reply:
done!